handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/BizkeeperHandler.java

```diff
@@ -74,6 +83,7 @@ public void handle(Invocation invocation, AsyncResponse asyncResp) {
     HystrixObservable<Response> command = delegate.createBizkeeperCommand(invocation);
 
     Observable<Response> observable = command.toObservable();
+    eventAlarm(invocation);
```

Comment:
need this logic every time ?
we can not know when will do the break action?

Reply:
yes，we can not know when will do the break action，the break action occurs in the hystrix code,so need this logic, every time.

Reply:
// record that we are returning a short-circuited fallback
        eventNotifier.markEvent(HystrixEventType.SHORT_CIRCUITED, commandKey);

when allowRequest return false, will invoke this, it seems that we no need to check everytime.
please check is it possible to optimize